tests/test_transactions.py

```diff
@@ -122,6 +125,7 @@ class TestTransactions(test.TruncationTestCase):
             team = await Team.create(name="Test team")
             await event.participants.add(team)
 
+    @test.skip("start_transaction is dodgy")
```

Comment:
I checked, we can delete `test_transaction_exception_` 1 & 3 as they are essentially duplicates of 2 & 4.

Reply:
removed, i renamed them to keep the numbering consistent as well

Reply:
:+1: